### ISSUE TITLE
gltfpack fails to open file arguments given as absolute paths

### ISSUE THREAD

**Issue by s3ththompson** [zeux/meshoptimizer#224]
I'm having difficulty getting gltfpack to open files that are given as absolute paths.

```console
$ gltfpack -i /private/var/folders/yk/05kl0s_d45jfc6hn2bz_h0km0000gn/T/input.gltf -o out.glb // temp file
Error loading /private/var/folders/yk/05kl0s_d45jfc6hn2bz_h0km0000gn/T/input.gltf: file not found

$ gltfpack -i ~/Downloads/input.gltf -o out.glb
Error loading /Users/seththompson/Downloads/input.gltf: file not found

$ gltfpack -i ./input.gltf -o out.glb
// Works
```

It looks like `gltfpack` just passes the input argument to `cgltf` so perhaps the issue is there? I did notice this issue https://github.com/jkuhlmann/cgltf/issues/94 where someone mentions "I had a platform issue with the fopen call..." which seemed like it might be related?

For what it's worth, I'm on macOS 11.0 (Big Sur), but I can't image how that's the issue.

**Comment by zeux** [comment]
Is that with a native binary or with the one from npm?

**Comment by s3ththompson** [comment]
It's the binary installed by gltfpack 0.15.1 on npm

**Comment by zeux** [comment]
I can reproduce this and this is specific to the npm build. I'll take a look at fixing this, in the meantime you can get a native build from the Releases page which shouldn't have this problem.

**Comment by zeux** [comment]
I've published gltfpack 0.15.2 to npm that has a fix for this bug.
